TACC/Core-CMS-Custom
# chore(apcd): WP-58 CMD-13 remove unused template

## Overview

This template is now not used. (It was a customization of a [retired template](https://github.com/TACC/Core-CMS/tree/v4.21.0/taccsite_cms/templates/guides).)

Pages that used it now use Core-CMS guide template and manual content.


## Related

- to abandon [WP-58](https://tacc-main.atlassian.net/browse/WP-58)
- to continue [CMD-143](https://tacc-main.atlassian.net/browse/CMD-143)
- supported by https://github.com/TACC/Core-Portal-Deployments/commit/8d9f2d6

## Changes

- **deleted** a template

## Testing & UI

1. Visit these pages as editor or admin:
    - https://dev.apcd.tacc.utexas.edu/help/getting-started/
    - https://apcd-qa.tacc.utexas.edu/help/getting-started/ 
    - https://txapcd.org/help/getting-started/
2. Verify each pages' content is manual i.e. editable.

> [!TIP]
> Different UI on [dev](https://dev.apcd.tacc.utexas.edu/help/getting-started/) because [TACC_CORE_STYLES_VERSION = 2](https://github.com/TACC/Core-Portal-Deployments/blob/a13bf7d/apcd/camino/cms.dev.settings_custom.py#L50-L51).
